CMakeLists.txt

```diff
@@ -205,6 +207,92 @@ if (NN_ENABLE_NANOCAT)
     target_link_libraries (nanocat ${PROJECT_NAME})
 endif ()
 
+# Build the documenation
+if (NN_ENABLE_DOC)
+    find_program (ASCIIDOCTOR_EXE asciidoctor)
+    if (NOT ASCIIDOCTOR_EXE)
+        message (SEND_ERROR "asciidoctor not found, cannot create docs")
+    endif ()
```

Comment:
`SEND_ERROR` continues execution, so the whole block below should be skipped if asciidoctor isn't found.